homeassistant/components/airvisual_pro/sensor.py

```diff
@@ -34,67 +36,93 @@
 SENSOR_KIND_TEMPERATURE = "temperature"
 SENSOR_KIND_VOC = "voc"
 
+
+@dataclass
+class AirVisualProMeasurementKeyMixin:
+    """Define an entity description mixin to include a measurement key."""
+
+    value_fn: Callable[[dict[str, Any], dict[str, Any], dict[str, Any]], float | int]
+
+
+@dataclass
+class AirVisualProMeasurementDescription(
+    SensorEntityDescription, AirVisualProMeasurementKeyMixin
+):
+    """Describe an AirVisual Pro sensor."""
+
+
 SENSOR_DESCRIPTIONS = (
-    SensorEntityDescription(
+    AirVisualProMeasurementDescription(
         key=SENSOR_KIND_AQI,
```

Comment:
Consider removing the constants for the keys since the constants are only used in one place.